### PR TITLE
Fix - Error "Parsing error: "parserOptions.project" has been set"

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will fix issue with "Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser." during eslint source check.

### CODE REVIEW

**Comment by webmaster128** [comment]
How is it possible to reproduce the problem? Excluding the story files from linting entirely looks like a big limtation.

Did you reload VS Code after https://github.com/iov-one/ponferrada/pull/1019 was merged? I had to do that such that the plugin picks up some config changes.

File: packages/bierzo-wallet/.eslintignore [hunk 1]
```diff
@@ -1 +1,3 @@
 build/
+types/
```
Comment (webmaster128) [inline packages/bierzo-wallet/.eslintignore]: This folder does not exist for the apps. It only exists for the libraries.

**Comment by albertandrejev** [comment]
> Did you reload VS Code after #1019 was merged? I had to do that such that the plugin picks up some config changes.

Yes I did, also this PR causes this issue . If you will scroll to the top of the any TS file you will see this error. At least Abel confirms this too.

> Excluding the story files from linting entirely looks like a big limtation.

I can include story files back, I did this to speedup `lint`'ing process.

**Comment by webmaster128** [comment]
> Yes I did, also this PR causes this issue . If you will scroll to the top of the any TS file you will see this error. At least Abel confirms this too.

Yeah, I had the problem too when I configured `parserOptions.project` for the first time. But after a `git clean -xdf && yarn install` and VS Code restart, the VS Code plugin I did not have any issues.

But I just found a way for me to reproduce: whenever I add a new file, this is not picked up by the IDE without reloading.

Bildschirmfoto 2020-02-17 um 23 23 05

After reload, no problem.

**Comment by webmaster128** [comment]
This is documented here

> There is one issue which will be fixed soon (IDE errors when creating a new file), but that is unrelated to this thread.

https://github.com/typescript-eslint/typescript-eslint/issues/967#issuecomment-539091676

See https://github.com/typescript-eslint/typescript-eslint/issues/864 fixed in https://github.com/typescript-eslint/typescript-eslint/pull/973, follow-up bug https://github.com/typescript-eslint/typescript-eslint/issues/1091 with fixes https://github.com/typescript-eslint/typescript-eslint/pull/1106 and https://github.com/typescript-eslint/typescript-eslint/pull/1111 and so on …

Upgrading `@typescript-eslint/*` to the latest version 2.20.0 seems to help.

**Pull request closed without merging**